### DISCUSSION TITLE
Starting a Vercel community within the Free & Open-Source Software Community at NSBM Green University, Sri Lanka 👋

### DISCUSSION THREAD

**Discussion by WasathTheekshana** [vercel/community#6555] [Idea]
![FOSS Logo Stripe - Light](https://github.com/vercel/community/assets/91784445/3312f5d4-328d-4b8c-929d-c5d0d9762962)

---

### Who are we🤔?
We, the FOSS Community of NSBM, which originated in 2013, has a reputation of being the first university based Free & Open-Source Software community in Sri Lanka as well as being the first community to initiate a Mozilla Campus Club within a university in Sri Lanka🦊.

<a href="https://github.com/fossnsbm/" target="_blank"><img src="https://img.shields.io/badge/GitHub%20-810D84" alt="foss-github" /></a>
<a href="https://www.fossnsbm.org/" target="_blank"><img src="https://img.shields.io/badge/fossnsbm.org%20-5CB747" alt="fossnsbm.org" /></a>
<a href="https://www.facebook.com/foss.nsbm" target="_blank"><img src="https://img.shields.io/badge/Our%20Facebook%20Page%20-0047FF" alt="foss-facebook" /></a>

### Why are we doing this💡? 
The main reason that we decided to start a Vercel community within the NSBM FOSS community is because our go-to web dev tools for the majority of our open-source projects are the products offered by Vercel 🖤. As there are currently no communities that represent Vercel in Sri Lanka, we wanted to create a community where student developers & experienced developers alike can get-together and talk about all things Vercel.

### Our goals 🚩
- To organize Hackathons, Buildathons... etc. using Vercel products like NextJS & v0 with help from the global Vercel community.
- To connect with experienced developers from the global Vercel community & share knowledge either physically or through virtual means.

<br>

> [!TIP]
> FYI😜, We would love it if you are able to send some of your cool Vercel swags our way so that we can give it away as gifts and prizes to the volunteers in our community. 

<br>

**Email Us**
**info@fossnsbm.org**

**Comment by ayodhyaw2002** [comment]
**This would be great  and helpfull..** 😎👍

**Comment by hercial61** [comment]
I wish the community all the best in their endeaavour
